packages/amplify-schema-validator/package.json

```diff
@@ -17,7 +17,8 @@
     "lint-staged": "lint-staged"
   },
   "devDependencies": {
-    "@size-limit/preset-small-lib": "^2.2.4",
+    "@size-limit/preset-small-lib": "6.0.4",
```

Comment:
Use version 6 instead of the latest (8) to avoid the following problem
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/20614187/229628432-1782e78e-aba1-4aa2-bb1e-7c0e6631b1db.png">